packages/graphback-datasync/package.json

```diff
@@ -43,11 +43,12 @@
     "@graphback/runtime-mongo": "0.14.0-alpha4",
     "@graphql-toolkit/common": "0.10.7",
     "@graphql-toolkit/core": "0.10.7",
+    "@graphql-tools/utils": "^6.0.10",
     "graphql-compose": "7.18.1",
     "graphql-metadata": "0.6.3",
     "graphql-subscriptions": "1.1.0",
     "mongodb": "3.5.9",
-    "pluralize": "8.0.0"
+    "pluralize": "^8.0.0"
```

Comment:
```suggestion
    "pluralize": "8.0.0"
```

Reply:
Ah, sorry about that, I miss that all the time :D